tests/server/app/test_processing.py

```diff
@@ -35,6 +37,13 @@
 
 class ProcessingTest(ServerTest):
 
+    def setUp(self) -> None:
+        super().setUp()
+        wait_for_server_startup = os.environ.get('WAIT_FOR_STARTUP',
+                                                 '0') == '1'
+        if wait_for_server_startup:
+            time.sleep(10)
+
```

Comment:
Don't repeat yourself.

Reply:
Sorry for this stupid code. I should not have started the PR at that stage.